### PR TITLE
CDD-1926: ingestion validation documentation

### PR DESCRIPTION
**Pull request by phill-stanley**
# Description

This PR includes documentation for ingestion validation

Fixes #CDD-1926

---

## Type of change

Please select the options that are relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [x] Tech debt item (this is focused solely on addressing any relevant technical debt)

---

# Checklist:

- [x] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] I have added tests at the right levels to prove my change is effective
- [ ] I have added screenshots or screen grabs where appropriate
- [ ] I have added docstrings in the correct style [(google)](https://google.github.io/styleguide/pyguide.html#38-comments-and-docstrings)

### CODE REVIEW

File: docs/ingestion_validation.md [hunk 1]
```diff
@@ -0,0 +1,184 @@
...
+## Ingestion validation rules
+
+## Metric 
+
+The metric must be constructed in the following way.
+`<topic>_<metric_group>_<metric_detail>`
+
+- The `<topic>` must match the provided `Topic` property, this comparison is case insensitive
+  - Eg: `influenza` and `Influenza` is a valid match.
+- The `<metric_group>` must match the provided `Metric Group`
```
Comment (A-Ashiq) [inline docs/ingestion_validation.md:17]: Probs worth saying that this is case-sensitive unlike the topic part

**Comment by A-Ashiq** [comment]
Looks great, thanks for this @phill-stanley

**Review by A-Ashiq — APPROVED** [review]

File: docs/ingestion_validation.md (hunk 1 above)
Comment (phill-stanley) [reply docs/ingestion_validation.md]: Good suggestion, I've included this.

Comment (zesh92) [reply docs/ingestion_validation.md]: Resolved conversation as @phill-stanley has made the change @A-Ashiq recommended.

**Pull request merged**